packages/pds/src/services/moderation/views.ts

```diff
@@ -233,6 +234,11 @@ export class ModerationViews {
         .selectFrom('moderation_report')
         .where('subjectType', '=', 'com.atproto.repo.strongRef')
         .where('subjectUri', '=', result.uri)
+        .innerJoin(
+          'did_handle',
+          'did_handle.did',
+          'moderation_report.subjectDid',
+        )
```

Comment:
Same thing here—I believe that we want to be able to serve reports for subjects that have been deleted, so we'll need to switch to a left join, and omit the `subjectRepoHandle` field when we don't have a matching `did_handle`.